Public/Src/Utilities/UnitTests/Executables/TestProcess/Operation.cs

```diff
@@ -853,10 +876,10 @@ private void DoEnumerateDir()
 
         private void DoCreateSymlink()
         {
-            string target = AdditionalArgs ?? FileOrDirectoryToString(Path);
+            string target = AdditionalArgs ?? PathAsString;
 
             // delete whatever might exist at the link location
-            var linkPath = FileOrDirectoryToString(LinkPath);
+            var linkPath = LinkPathAsString;
             FileUtilities.DeleteFile(linkPath);

```

Comment:
This asserts that linkPath is non null/empty and should be conditionalized to only delete if a file exists at `linkPath`.